tests/integration/docusaurus/expectations/expectation_management.py

```diff
@@ -0,0 +1,56 @@
+# setup
+import great_expectations as gx
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.expectations.core import (
+    ExpectColumnValuesToBeInSet,
+    ExpectColumnValuesToNotBeNull,
+)
+
+context = gx.get_context()
+suite_name = "test-suite"
+column_a = "a"
+column_b = "b"
+column_c = "c"
+
+# create a suite
+suite = ExpectationSuite(name=suite_name)
+suite = context.suites.add(suite=suite)
+
+
+# add expectations
+expectation_a = ExpectColumnValuesToBeInSet(
```

Comment:
Non-blocking / very nit: I think this may be 2% easier to read if it were just `suite.add_expectation(ExpectColumnValuesToBeInSet(column=column_a, value_set=["foo", "bar", "baz"]))`. My thinking here is:
* the assignment makes me wonder if this will be referenced again below
* I think 1 line is just a bit less to look at, assuming it's not too terse.

Reply:
I think we should use `gxe.Expect_` if this is going to be referenced in docs (I believe that is our canonical pattern?)

Reply:
thanks both! updated in [018c3a3](https://github.com/great-expectations/great_expectations/pull/9213/commits/018c3a343dbbefc7dbc5c2359be124e73e82bc84)